scripts/lib.sh

```diff
@@ -172,29 +175,38 @@ function start_lightningd() {
   fi
   lightningd $LIGHTNING_FLAGS --lightning-dir=$FM_CLN_DIR --plugin=$FM_BIN_DIR/gateway-cln-extension &
   echo $! >> $FM_PID_FILE
+  echo "started lightningd"
 }
 
 function start_lnd() {
+  echo "starting lnd"
   await_bitcoin_rpc
   lnd --lnddir=$FM_LND_DIR &
   echo $! >> $FM_PID_FILE
+  echo "started lnd"
 }
 
 function start_gatewayd() {
+  echo "starting gatewayd"
   await_fedimint_block_sync
   $FM_BIN_DIR/gatewayd &
   echo $! >> $FM_PID_FILE
   gw_connect_fed
+  echo "started gatewayd"
 }
 
 function start_electrs() {
+  echo "starting electrs"
   await_bitcoin_rpc
   electrs --conf-dir "$FM_ELECTRS_DIR" --db-dir "$FM_ELECTRS_DIR" --daemon-dir "$FM_BTC_DIR" &
   echo $! >> $FM_PID_FILE
+  echo "started electrs"
```

Comment:
for future optimization: should we make this a [done] at he end of the line? The first echo would just not print a newline.

Reply:
https://github.com/fedimint/fedimint/issues/1882